sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/checkin/SentryCheckInAdvice.java

```diff
@@ -51,7 +55,20 @@ public Object invoke(final @NotNull MethodInvocation invocation) throws Throwabl
     }
 
     final boolean isHeartbeatOnly = checkInAnnotation.heartbeat();
-    final @Nullable String monitorSlug = checkInAnnotation.value();
+
+    @Nullable String monitorSlug = checkInAnnotation.value();
+
+    if (resolver != null) {
+      try {
+        monitorSlug = resolver.resolveStringValue(checkInAnnotation.value());
+      } catch (Throwable e) {
+        hub.getOptions()
```

Comment:
`m` can we also include the exception in the log so it's easier for users to figure out the problem?